thoth/glyph/lib.py

```diff
@@ -92,39 +93,42 @@ def classify_messages(messages: list, model: str):
         _LOGGER.error("No commits found!")
         return
 
-    df = pd.DataFrame(messages, columns=["message"])
-    df = df.replace("\n", "", regex=True)
-    commits = list(df["message"].astype(str))
     if model is None:
         _LOGGER.info("Using default model")
-        model = DEFAULT_MODEL_PATH
-    elif not os.path.exists(model):
-        _LOGGER.error("Model not found, using default model instead")
-        model = DEFAULT_MODEL_PATH
-
-    _LOGGER.info("Model Path : " + model)
-    classifier = load_model(model)
-    labels = classifier.predict(commits)
-    res = list(zip(*labels))
-    res_list = [x[0] for x in res]
-    lst2 = [item[0] for item in res_list]
-    df["labels_predicted"] = lst2
-    _LOGGER.info(str(len(messages)) + " commits classified")
-    return df
+        model = DEFAULT_MODEL
+
+    if model == FASTTEXT_MODEL:
+        return FasttextModel.classify_messages(messages)
 
 
 def classify_message(message: str, model: str) -> str:
     if message is None or message.strip() == "":
         raise NoMessageEnteredException
+
     if model is None:
         _LOGGER.info("Using default model")
-        model = DEFAULT_MODEL_PATH
-    elif not os.path.exists(model):
-        raise ModelNotFoundException
-
-    _LOGGER.info("Model Path : " + model)
-    classifier = load_model(model)
-    label = classifier.predict(message.lower())
-    label_string = str(label[0][0])[9:]
-    _LOGGER.info("Label : " + label_string)
-    return label_string
+        model = DEFAULT_MODEL
+
+    if model == FASTTEXT_MODEL:
+        return FasttextModel.classify_message(message)
+
+
+def generate_log(messages: list, format: str, model: str):
+    df = classify_messages(messages, model)
+
+    keys = ["features", "corrective", "perfective", "nonfunctional", "unknown"]
```

Comment:
Might be a good idea to rename these to some commonly used.

Reply:
Fixed it for this specific model. Ideally this kind of translation should reside inside the model class itself.
Have added a TODO to improve it later.
For now, it will run as per expectation (with commonly used terms such as Bug-fixes, improvements etc.)